CHANGELOG.md

```diff
@@ -124,6 +124,7 @@
 * [#4354](https://github.com/bbatsov/rubocop/pull/4354): Add autocorrect to `Style/FormatString`. ([@hoshinotsuyoshi][])
 * [#4021](https://github.com/bbatsov/rubocop/pull/4021): Add new `Style/MultipleComparison` cop. ([@dabroz][])
 * New `Lint/RescueType` cop. ([@rrosenblum][])
+* [#4396](https://github.com/bbatsov/rubocop/pull/4396): Add new `Rails/IndexTrue` cop. ([@alexcstark][])
```

Comment:
This has to be moved to the proper section of the changelog.

Reply:
Ah I see - this needs to be with the uncommitted changes and release.